Sources/Hummingbird/Router/Parameters+UUID.swift

```diff
@@ -33,7 +33,7 @@ extension Parameters {
         guard let param = self[s[...]],
               let result = UUID(uuidString: String(param))
         else {
-            throw HTTPError(.badRequest)
+            throw HTTPError(.badRequest, message: "Parameter '\(s)' can not be converted to the expected type (UUID)")
```

Comment:
We shouldn't expose the parameter name here, that's internal information. We can however expose the value provided. I'm also not sure we should provide the expected type per-say.